package/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN yum-config-manager --enable ol7_optional_latest && \
     yum update -y && \
     # NOTE - psmisc is needed for "killall" command, which Rancher uses to kill the proxy container on helm operation pods
     yum install -y bash bash-completion gzip jq tar unzip vim wget psmisc && \
-    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne14/x86_64 && \
-    yum -y install kubectl-1.21.6-2.el7.x86_64 && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64 && \
+    yum -y install kubectl-1.24.8-1.el7.x86_64 && \
```

Comment:
Are there any issues with k8s version skew here? Technically this would only support k8s 1.23, 1.24, 1.25. Does VZ 1.5 support k8s 1.22? I guess if all of the upgrade pipelines work then it should be OK?